pkg/kubelet/kubelet_volumes_test.go

```diff
@@ -504,7 +505,9 @@ func TestVolumeUnmountAndDetachControllerEnabled(t *testing.T) {
 		1 /* expectedSetUpCallCount */, testKubelet.volumePlugin))
 
 	// Remove pod
+	kubelet.podWorkers.(*fakePodWorkers).statusLock.Lock()
 	kubelet.podWorkers.(*fakePodWorkers).removeRuntime = map[types.UID]bool{pod.UID: true}
+	kubelet.podWorkers.(*fakePodWorkers).statusLock.Unlock()
 	kubelet.podManager.SetPods([]*v1.Pod{})
```

Comment:
L319-321 has the same code without lock and a TODO, can we put them together into a method like:

https://github.com/kubernetes/kubernetes/blob/5e85b68ea848b7c736975a9ef7458f71bd87e6de/pkg/kubelet/pod_workers_test.go#L103-L107